bumble/controller.py

```diff
@@ -194,6 +194,21 @@ def __init__(
 
         self.terminated = asyncio.get_running_loop().create_future()
 
+    @classmethod
```

Comment:
It's maybe too verbose to have a `create` method with all the same args as `__init__`.
A generic solution would be cleaner.
Something like:
```
async def async_call(function, *args, **kwargs):
    return function(*args, **kwargs)
```
(which could be put in `utils.py`)
Can be invoked either with an argument-less lambda, or passing the func and its args. Examples:
```
c = await async_call(Controller, name, ...)
```
or
```
c = await async_call(lambda: Controller(name, ...))
```

Or if you actually need to create a function object, this decorator would be convenient:
```
def asyncify(function):
    return partial(async_call, function)
```
which you'd use like:
```
constructor = asyncify(Controller)
...
controller = await constructor(name, ...)
```

Reply:
Excellent idea.

I've moved forward with your first suggested solution, and it works great.

I suppose creating a function object would make it even cleaner on the Rust side, but I'm only using `async_call` in one place, so I've only added minimum necessary.

Reply:
Thinking about it further, I decided that its worth having a cleaner solution, so I also added the "asyncify" function/decorator.